exercise-src/react-native/06-custom-components/01-problem/App.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from "react"
+import { SafeAreaView, ScrollView, Text, View } from "react-native"
+
+const states = [
+  {
+    name: "Illinois",
+    short: "IL",
+  },
+  {
+    name: "Wisconsin",
+    short: "WI",
+  },
+]
+
+export const StateList: FC = () => {}
```

Comment:
Is this too handhold-y for the exercise? I haven't written the exercise directions yet.
A few other cases of this

Reply:
To make the diff better, could we have a `return` statement in here? If we need to, we can return a blank fragment.